Resources/Prototypes/DeltaV/Entities/Objects/Devices/station_beacon.yml

```
@@ -1,113 +1,113 @@
   id: DefaultStationBeaconVirology
   suffix: Virology
   components:
   - type: NavMapBeacon
-    text: station-beacon-virology
+    defaultText: station-beacon-virology
 
 - type: entity
-  parent: DefaultStationBeaconMedical
+  parent: DefaultStationBeacon
   id: DefaultStationBeaconMetempsychosis
   suffix: Cloning #Metempsychosis
```

yamllint

[warning] too few spaces before comment

(comments)